### PR TITLE
Hungarian translation (work in progress)

### PR DESCRIPTION
**Pull request by anon569**

### CODE REVIEW

File: data/language/hu-HU.txt [hunk 1]
```diff
@@ -1,17 +1,110 @@
...
+STR_0033    :Ismeretlen bódé (1F)
+STR_0034    :Bolt
+STR_0035    :Körhinta
+STR_0036    :Ismeretlen bódé (22)
+STR_0037    :Információs bódé
+STR_0038    :WC
+STR_0039    :Óriáskerék 
+STR_0040    :Mozgás-szimulátor
+STR_0041    :3D mozi
+STR_0042    :Top Spin
```
Comment (Broxzier) [inline data/language/hu-HU.txt]: Missed this one, or is it the same in English?

File: data/language/hu-HU.txt [hunk 2]
```diff
@@ -1,17 +1,110 @@
...
+STR_0043    :Űrgyűrűk 
+STR_0044    :Fordított szabadeséses hullámvasút
+STR_0045    :Lift
```
Comment (Broxzier) [inline data/language/hu-HU.txt]: English leftover?

File: data/language/hu-HU.txt [hunk 3]
```diff
@@ -1,17 +1,110 @@
...
+STR_0060    :Ismeretlen attrakció (3A)
+STR_0061    :Virginia Reel
+STR_0062    :Csobbanó csónakok
+STR_0063    :Mini helikopterek
+STR_0064    :Fekvő hullámvasút
 STR_0065    :Felfüggesztett egysínű
+STR_0066    :Ismeretlen attrakció (40)
+STR_0067    :Irányváltó hullámvasút
+STR_0068    :Szívvonalcsavró hullámvasút
+STR_0069    :Minigolf
```
Comment (Broxzier) [inline data/language/hu-HU.txt]: And this one?

File: data/language/hu-HU.txt [hunk 4]
```diff
@@ -168,14 +494,42 @@ STR_2001    :Pizza
...
 STR_2013    :Limonádé
 STR_2014    :Üres doboz
 STR_2015    :Üres flakon
+STR_2016    :Léggömbök
+STR_2017    :Plüssállatok
+STR_2018    :Parktérképek
+STR_2020    :Esernyők
+STR_2021    :Italok
+STR_2027    :Szemét
+STR_2031    :Popcorn
```
Comment (Broxzier) [inline data/language/hu-HU.txt]: English leftover?

File: data/language/hu-HU.txt [hunk 5]
```diff
@@ -1,17 +1,110 @@
...
+STR_0074    :Egysínű biciklik
+STR_0075    :Kompakt fordított hullámvasút
+STR_0076    :Vízi hullámvasút 
+STR_0077    :Levegővel működtetett függőleges hullámvasút
+STR_0078    :Fordított hajtű hullámvasút
+STR_0079    :Varázsszőnyeg
+STR_0080    :Tengeralattjáró-utazás
+STR_0081    :Folyami tutajok
+STR_0082    :Ismeretlen attrakció (50)
+STR_0083    :Enterprise
```
Comment (Broxzier) [inline data/language/hu-HU.txt]: English leftover?

File: data/language/hu-HU.txt [hunk 6]
```diff
@@ -198,26 +557,455 @@ STR_2340    :Líra (L)
...
+STR_2528    :???
+STR_2529    :???
+STR_2530    :???
+STR_2531    :???
+STR_2532    :???
+STR_2533    :Backspace
+STR_2534    :Tab
+STR_2535    :???
+STR_2536    :???
+STR_2537    :Clear
```
Comment (Broxzier) [inline data/language/hu-HU.txt]: English leftover?

File: data/language/hu-HU.txt [hunk 7]
```diff
@@ -198,26 +557,455 @@ STR_2340    :Líra (L)
...
+STR_2557    :Szóköz
+STR_2558    :PgUp
+STR_2559    :PgDn
+STR_2560    :End
+STR_2561    :Home
+STR_2562    :Balra nyíl
+STR_2563    :Felfelé nyíl
+STR_2564    :Jobbra nyíl
+STR_2565    :Lefelé nyíl
+STR_2566    :Select
```
Comment (Broxzier) [inline data/language/hu-HU.txt]: English leftover?

File: data/language/hu-HU.txt [hunk 8]
```diff
@@ -198,26 +557,455 @@ STR_2340    :Líra (L)
...
+STR_2567    :Print
```
Comment (Broxzier) [inline data/language/hu-HU.txt]: English leftover?

File: data/language/hu-HU.txt [hunk 9]
```diff
@@ -198,26 +557,455 @@ STR_2340    :Líra (L)
...
+STR_2568    :Execute
```
Comment (Broxzier) [inline data/language/hu-HU.txt]: English leftover?

File: data/language/hu-HU.txt [hunk 10]
```diff
@@ -168,14 +494,42 @@ STR_2001    :Pizza
...
+STR_2216    :{WINDOW_COLOUR_2}{COMMA16}{DEGREE}C
+STR_2217    :{WINDOW_COLOUR_2}{COMMA16}{DEGREE}F
+STR_2220    :{WINDOW_COLOUR_2}Park értékelése: {BLACK}{COMMA16}
+STR_2221    :{SMALLFONT}{BLACK}Park értékelése: {COMMA16}
+STR_2222    :{SMALLFONT}{BLACK}{STRINGID}
+STR_2233    :{SMALLFONT}{BLACK}Park információ
+STR_2234    :Legutóbbi üzenetek
+STR_2235    :{SMALLFONT}{STRINGID} {STRINGID}
 STR_2236    :Január
 STR_2237    :Februáy
```
Comment (Broxzier) [inline data/language/hu-HU.txt]: This should be 'Február', could you update it?

File: data/language/hu-HU.txt [hunk 11]
```diff
@@ -198,26 +557,455 @@ STR_2340    :Líra (L)
...
+STR_5194    :Víz
+STR_5217    :Csalások
+STR_5218    :Témák
+STR_5222    :Betöltés/mentés
+STR_5236    :Ablak
+STR_5244    :Témák
+STR_5260    :Képmentés a teljes térképről
+STR_5298    :{RED}{STRINGID}
+STR_5299    :{LIGHTPINK}{STRINGID}
+STR_5304    :Title Sequence:
```
Comment (Broxzier) [inline data/language/hu-HU.txt]: English leftover?

File: data/language/hu-HU.txt [hunk 12]
```diff
@@ -198,26 +557,455 @@ STR_2340    :Líra (L)
...
+STR_5306    :RollerCoaster Tycoon 1 (AA)
+STR_5307    :RollerCoaster Tycoon 1 (AA + LL)
+STR_5308    :RollerCoaster Tycoon 2
+STR_5309    :OpenRCT2
+STR_5310    :Véletlenszerű
+STR_5334    :Jég
+STR_5344    :Változási napló
+STR_5349    :{SMALLFONT}{BLACK}Minden játék
+STR_5350    :Max
+STR_5351    :Min
```
Comment (Broxzier) [inline data/language/hu-HU.txt]: English leftover?

File: data/language/hu-HU.txt [hunk 13]
```diff
@@ -198,26 +557,455 @@ STR_2340    :Líra (L)
...
 
-# This language is unmaintained. All untranslated strings have been removed. 
-# If you intend to become the maintainer, please copy the missing strings from en-GB.txt and translate them.
+#tooltips
+STR_5877    :Szoftveres
+STR_5879    :OpenGL (kísérleti)
+STR_5887    :Előtag
+STR_5888    :Utótag
+STR_5910    :Alkalmazás
+STR_5913    :Chat
```
Comment (Broxzier) [inline data/language/hu-HU.txt]: English leftover?

File: data/language/hu-HU.txt [hunk 14]
```diff
@@ -1,17 +1,110 @@
...
+STR_0019    :Bányavonat hullámvasút
 STR_0020    :Libegő
-STR_0054    :Fa hullámvasút
-STR_0056    :Vad egér
+STR_0021    :Dugóhúzó hullámvasút
+STR_0022    :Útvesztő
+STR_0023    :Spirál csúszda
+STR_0024    :Gokart
+STR_0025    :Faúsztató
+STR_0026    :Folyami gyorslányok
```
Comment (Broxzier) [inline data/language/hu-HU.txt]: This translates to 'Fluffy fast girls', and searching for it doesn't result in things related to river rapids.

My Hungarian friend (Morinas) mentions that there is no specific word for this ride type, and gave some suggestions:
- Vadvizi Evezés
- Vadvizi Kaland

I trust you can better decide that me which would be the best option here.

Comment (anon569) [reply data/language/hu-HU.txt]: I changed it to Vadvízi evezés.

File: data/language/hu-HU.txt (hunk 2 above)
Comment (anon569) [reply data/language/hu-HU.txt]: It's the same in English.

File: data/language/hu-HU.txt [hunk 15]
```diff
@@ -856,7 +863,7 @@ STR_5244    :Témák
 STR_5260    :Képmentés a teljes térképről
 STR_5298    :{RED}{STRINGID}
 STR_5299    :{LIGHTPINK}{STRINGID}
-STR_5304    :Title Sequence:
+STR_5304    :Címképernyős  játékpillanatok:
```
Comment (Broxzier) [inline data/language/hu-HU.txt]: There are two spaces here

File: data/language/hu-HU.txt [hunk 16]
```diff
@@ -198,26 +564,455 @@ STR_2340    :Líra (L)
...
+STR_6229    :{WINDOW_COLOUR_2}{STRINGID}: {STRINGID}
+STR_6230    :{STRINGID}:
+STR_6231    :{WINDOW_COLOUR_2}{STRINGID}: {MOVE_X}{185}{STRINGID}
+STR_6236    :Játékosok
+STR_6237    :Csoportok
+STR_6259    :Letiltva
+
+#############
+# Scenarios #
+################
```
Comment (Gymnasiast) [inline data/language/hu-HU.txt]: Do you plan to translate the scenarios eventually as well?

Comment (anon569) [reply data/language/hu-HU.txt]: Yes. I want to translate everything.

File: data/language/hu-HU.txt (hunk 15 above)
Comment (anon569) [reply data/language/hu-HU.txt]: Thanks. I fixed it.

File: data/language/hu-HU.txt [hunk 17]
```diff
@@ -98,34 +436,204 @@ STR_1198    :Baleset!
...
+STR_1645    :
+STR_1646    :
+STR_1647    :
+STR_1675    :{POP16}{VELOCITY}
+STR_1699    :Túl sok ember van a játékban
 STR_1700    :Új mindenes felvétele
 STR_1701    :Új szerelő felvétele
 STR_1702    :Új biztonsági őr felvétele
 STR_1703    :Új szórakoztató felvétele
-STR_1712    :{INLINE_SPRITE}{247}{19}{00}{00}{WINDOW_COLOUR_2}Utak takarítása
```
Comment (Gymnasiast) [inline data/language/hu-HU.txt]: It looks like you removed strings 1712-1715. Is that intentional?

File: data/language/hu-HU.txt [hunk 18]
```diff
@@ -251,32 +251,168 @@ STR_0970    :+450{DEGREE}
...
-STR_1011    :Összes bezárása
-STR_1012    :Összes megnyitása
-STR_1013    :Park bezárása
-STR_1014    :Park megnyitása
+STR_0990    :{SMALLFONT}{BLACK}Építkezés
+STR_0991    :Állomáselem
+STR_0992    :{SMALLFONT}{BLACK}A játék/épület teljes lerombolása
+STR_0993    :Játék/épület lerombolása
+STR_0994    :Lerombolás
+STR_0995    :{WINDOW_COLOUR_1}Biztosan le akarod bontani ezt:{STRINGID}?
```
Comment (Broxzier) [inline data/language/hu-HU.txt]: Missing space after the colon

File: data/language/hu-HU.txt (hunk 17 above)
Comment (anon569) [reply data/language/hu-HU.txt]: Thanks, fixed.

File: data/language/hu-HU.txt (hunk 18 above)
Comment (anon569) [reply data/language/hu-HU.txt]: Thanks, fixed.

File: data/language/hu-HU.txt [hunk 19]
```diff
@@ -809,23 +809,108 @@ STR_2012    :Sült csirke
...
-STR_2018    :Parktérképek
-STR_2020    :Esernyők
-STR_2021    :Italok
+STR_2016    :Lufi
+STR_2017    :Plüssjáték
+STR_2018    :Parktérkép
+STR_2019    :Utazás közbeni fotó
+STR_2020    :Esernyő
+STR_2021    :Innivaló
+STR_2022    :Hamburger
```
Comment (Broxzier) [inline data/language/hu-HU.txt]: Possibly an English leftover?

File: data/language/hu-HU.txt [hunk 20]
```diff
@@ -809,23 +809,108 @@ STR_2012    :Sült csirke
...
+STR_2023    :Sült krumpli
+STR_2024    :Jégkrém
+STR_2025    :Vattacukor
+STR_2026    :Üres italos doboz
 STR_2027    :Szemét
+STR_2028    :Üres hamburgerdoboz
+STR_2029    :Pizza
```
Comment (Broxzier) [inline data/language/hu-HU.txt]: Possibly an English leftover?

File: data/language/hu-HU.txt [hunk 21]
```diff
@@ -809,23 +809,108 @@ STR_2012    :Sült csirke
...
+STR_2030    :Kupon
 STR_2031    :Pattogatott kukorica
+STR_2032    :Hot dog
```
Comment (Broxzier) [inline data/language/hu-HU.txt]: Possibly an English leftover?

File: data/language/hu-HU.txt (hunk 19 above)
Comment (anon569) [reply data/language/hu-HU.txt]: Yeah, it's the same in Hungarian.

File: data/language/hu-HU.txt (hunk 20 above)
Comment (anon569) [reply data/language/hu-HU.txt]: Same in Hungarian.

File: data/language/hu-HU.txt (hunk 21 above)
Comment (anon569) [reply data/language/hu-HU.txt]: Same in Hungarian.

**Pull request merged**